Python/sysmodule.c

```diff
@@ -3358,6 +3358,16 @@ make_impl_info(PyObject *version_info)
     if (res < 0)
         goto error;
 
+#ifdef VPLATFORM
+    value = PyUnicode_FromString(VPLATFORM);
+    if (value == NULL)
+        goto error;
+    res = PyDict_SetItemString(impl_info, "_architecture", value);
```

Comment:
sys.implementation already has "_multiarch" attribute, so maybe just "arch" (instead of "architecture") would be more consistent.

Reply:
I also think `arch` is more concise and accurately describes the intent (maybe it should satisfy Zooba?).